starknet_py/common.py

```diff
@@ -31,6 +40,23 @@ def int_from_hex(number: Union[str, int]) -> int:
     return number if isinstance(number, int) else int(number, 16)
 
 
+def get_selector_from_name(func_name: str) -> int:
+    """
+    Returns the selector of a contract's function name.
+    """
+    if func_name in [DEFAULT_ENTRY_POINT_NAME, DEFAULT_L1_ENTRY_POINT_NAME]:
+        return DEFAULT_ENTRY_POINT_SELECTOR
+
+    return starknet_keccak(data=func_name.encode("ascii"))
+
+
+def starknet_keccak(data: bytes) -> int:
+    """
+    A variant of eth-keccak that computes a value that fits in a StarkNet field element.
+    """
+    return int_from_bytes(keccak(data)) & MASK_250
```

Comment:
Perhaps at least this one should be kept private?

Reply:
Definitely could.